### PR TITLE
[move-prover] Added the CLI flag `only-verify-spec` for Prover

### PR DESCRIPTION
**Pull request by junkil-park**
- Added the CLI flag `only-verify-spec` for Prover
    - To only verify the functions with spec defined,
    - thus skipping verifying the functions without specs
    - for example, `cargo run -- --only-verify-spec move_filename.move`
- Removed `old_libra_coin.move` which has been replaced by `libra.move`

## Motivation

To add a new flag for Prover

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

cargo test

### CODE REVIEW

File: language/move-prover/src/cli.rs [hunk 1]
```diff
@@ -143,6 +146,11 @@ impl Options {
                     .long("generate-only")
                     .help("only generate boogie file but do not call boogie"),
             )
+            .arg(
+                Arg::with_name("only-verify-spec")
+                    .long("only-verify-spec")
+                    .help("whether to only verify the functions having their specification defined"),
```
Comment (wrwg) [inline language/move-prover/src/cli.rs:152]: Perhaps change to "whether to only verify functions which have associated specifications"

Comment (junkil-park) [reply language/move-prover/src/cli.rs]: Changed.

**Pull request closed without merging**